.github/workflows/build_image.yaml

```diff
@@ -0,0 +1,18 @@
+
+name: Build-Kubeflow-Triage
+on: [push]
+
+jobs:
+  pre-build-container:
+      runs-on: ubuntu-latest
+      steps:
+      - uses: actions/checkout@master
+      - name: Build Image
+        run: |
+          cd $GITHUB_WORKSPACE
+          echo ${PASSWORD} | docker login -u $USERNAME --password-stdin
+          docker build -t hamelsmu/kubeflow-triage -f Issue_Triage/Dockerfile .
```

Comment:
So this is currently pushing to your Docker registry? Could you add a TODO to change this to a Kubeflow GCR registry?

Reply:
done